xlsx/src/export/worksheets.rs

```diff
@@ -280,6 +288,16 @@ pub(crate) fn get_worksheet_xml(
         }
     }
 
+    let merge_cells_section = if merged_cells_count > 0 {
+        format!(
+            "<mergeCells count=\"{merged_cells_count}\">{merged_cells}</mergeCells>",
+            merged_cells_count = merged_cells_count,
+            merged_cells = merged_cells
+        )
+    } else {
+        String::new()
```

Comment:
This is correct but `"".to_string()` is what we use elsewhere.

Reply:
Done